mediorum/server/serve_blob.go

```diff
@@ -47,6 +50,9 @@ func (ss *MediorumServer) getBlob(c echo.Context) error {
 		c.Response().Header().Set("Content-Disposition", "attachment; filename="+contentDisposition)
 	}
 
+	// v1 feature parity
+	go ss.logTrackListen(c)
```

Comment:
probably should handle deduping requests based on requester+track because streaming a single track can cause multiple requests which return 206 Partial Content to get different byte ranges, especially when scrubbing or jumping around. but v1 doesn't do this anyway